rts/lra/lra-test/lra-test-arquillian-extension/src/main/java/io/narayana/lra/arquillian/NarayanaLRABaseUrlProvider.java

```diff
@@ -49,7 +49,7 @@ public Object lookup(ArquillianResource resource, Annotation... qualifiers) {
             return new URL(String.format("http://%s:%s", hostname, port));
         } catch (MalformedURLException murle) {
             throw new IllegalStateException(String.format("Cannot compose URL from host '%s' and port '%s' "
-                    + "provided with testsuite parameters 'lra.tck.suite.host' and 'lra.tck.suite.port'.", hostname, port), murle);
+                + "provided with testsuite parameters 'lra.tck.suite.host' and 'lra.tck.suite.port'.", hostname, port), murle);
```

Comment:
Hi @xstefank last comment I think :) It looks like a formatting only change? If you can revert that line it looks ready to merge to me

Reply:
Thanks for fixing